index.css

```diff
@@ -0,0 +1,20 @@
+body {
```

Comment:
- [ ] Kindly consider styling your page according to the Figma design including the colors, margin/padding and align-items using flexbox. so your design is completed 

|Your design|Figma design|
|-|-|
|![Screenshot_20220528_171403](https://user-images.githubusercontent.com/25227022/170833739-fd73dd66-63f5-407d-af0f-48fecb98314f.png)|![Screenshot_20220528_171759](https://user-images.githubusercontent.com/25227022/170833774-3d680e0d-e3fd-4c3a-9778-533bb2971e24.png)|